src/sqlancer/duckdb/gen/DuckDBInsertGenerator.java

```diff
@@ -41,7 +41,7 @@ private SQLQueryAdapter generate() {
     }
 
     @Override
-    protected void insertValue(DuckDBColumn tiDBColumn) {
+    protected void insertValue(DuckDBColumn aDuckDBColumn) {
```

Comment:
use `duckDBColumn` or `columnDuckDB` would be preferred

Reply:
`duckDBColumn` can not pass the CI as described https://github.com/sqlancer/sqlancer/pull/838#issue-1773263162

let's use `columnDuckDB `